### DISCUSSION TITLE
Why would you name it APACHE superset?

### DISCUSSION THREAD

**Discussion by butching** [apache/superset#22864] [Ideas]
Why would you name it APACHE superset if in your documentation you will endorse gunicorn and not the APACHE mod_wsgi?

**Comment by EugeneTorap** [comment]
Hi @butching! `APACHE` is the `Apache Software Foundation`.
Superset graduated from the incubator program and became a top-level project at the [Apache Software Foundation](https://en.wikipedia.org/wiki/Apache_Software_Foundation) in 2021.
https://en.wikipedia.org/wiki/Apache_Superset
